.eslintrc.js

```diff
@@ -1,6 +1,114 @@
 module.exports = {
-    "extends": "airbnb-base",
-    rules:{
-        "linebreak-style": 0
-    }
-};
\ No newline at end of file
+	"extends": [
+		"airbnb",
+		"prettier",
+		"prettier/react"
```

Comment:
Do we need React eslint rules in this project? My understanding was that it's just an express server sending over data.